src/main/java/org/example/service/TableService.java

```diff
@@ -0,0 +1,36 @@
+package org.example.service;
+
+import org.example.model.Developer;
+
+import java.io.PrintWriter;
+import java.util.List;
+
+public class TableService {
+
+    private static final List<Developer> developers = DeveloperService.getInstance().getDevelopers();
+
+    private static TableService INSTANCE = null;
```

Comment:
Remove null.

Check the Java conventions for naming:

- variables
- methods
- classes
- constant variables
- class files

Reply:
fix